packages/plugin-core/src/_extension.ts

```diff
@@ -580,6 +583,25 @@ export async function _activate(
           vscode.ConfigurationTarget.Global
         );
       }
+
+      // For new users, we want to load graph with new graph themes as default
+      let graphTheme;
+      const ABUserGroup = GRAPH_THEME_TEST.getUserGroup(
+        SegmentClient.instance().anonymousId
+      );
+      switch (ABUserGroup) {
+        case GraphThemeTestGroups.monokai: {
+          graphTheme = GraphThemeEnum.Monokai;
+          break;
+        }
+        case GraphThemeTestGroups.block: {
+          graphTheme = GraphThemeEnum.Block;
+          break;
+        }
+        default:
+          graphTheme = GraphThemeEnum.Classic;
+      }
+      MetadataService.instance().setGraphTheme(graphTheme);
```

Comment:
Hmm - for analytics purposes, it might be easier for us to analyze if we set a marker right here as well.  Maybe we can reuse the same event of changing graph themes, but add a property to the effect of "SetDuringInstall: true" or something like that. 

Otherwise, we'll need to remember that only folks who installed on v92 or greater will have had this altered.